### PR TITLE
Kd e2e scenarious 012 013

### PR DESCRIPTION
**Pull request by KostiantynDrozd-okta**
Added scenarios 0.1.2 0.1.3.
Updated imports to use direct cucumber instead of old cukes:cucumber.
Added closing browser after each scenario.

### CODE REVIEW

**Review by arvindkrishnakumar-okta — APPROVED** [review]

File: direct-auth-samples/src/test/java/pages/RootPage.java [hunk 1]
```diff
@@ -64,4 +67,24 @@ public boolean isLoginButtonDisplayed() {
...
+
+    public void waitForLoginButtonDisplayed(){
+        new WebDriverWait(driver, 5)
+                .until(ExpectedConditions.visibilityOf(logoutButton));
+
+    }
+
+    public boolean elementIsDisplayed(WebElement element){
+        try {
+        element.isDisplayed();
```
Comment (VitaliiTytarenko-okta) [inline direct-auth-samples/src/test/java/pages/RootPage.java:83]: Can `isDisplayed()` method return **false** without exception?..

File: direct-auth-samples/src/test/java/info/seleniumcucumber/userStepDefinitions/Login.java [hunk 2]
```diff
@@ -133,6 +133,34 @@ public void redirect_to_sspr_view() {
...
+		loginPage.passwordInput.sendKeys(PASSWORD);
+		loginPage.signInButton.click();
+	}
+
+	@Then("Mary sees a table with the claims from the userinfo response")
+	public void mary_sees_a_table_with_the_claims_from_the_userinfo_response() {
+		Assert.assertTrue(rootPage.idToken.isDisplayed());
+		Assert.assertTrue(rootPage.refreshToken.isDisplayed());
+		Assert.assertTrue(rootPage.email.isDisplayed());
+		System.out.println();
```
Comment (VitaliiTytarenko-okta) [inline direct-auth-samples/src/test/java/info/seleniumcucumber/userStepDefinitions/Login.java:154]: nit: redundant println()

File: direct-auth-samples/src/test/java/pages/RootPage.java (hunk 1 above)
Comment (KostiantynDrozd-okta) [reply direct-auth-samples/src/test/java/pages/RootPage.java:83]: @VitaliiTytarenko-okta 
in this case element throws NoSuchElementException.

Comment (VitaliiTytarenko-okta) [reply direct-auth-samples/src/test/java/pages/RootPage.java:83]: From the JavaDoc:
```
Is this element displayed or not? This method avoids the problem of having to parse an element's "style" attribute.
Returns: Whether or not the element is displayed
```
Therefore, I believe that it **can** return `true/false` without exceptions.

Comment (VitaliiTytarenko-okta) [reply direct-auth-samples/src/test/java/pages/RootPage.java:83]: Another question: what will your method return if `isDisplayed()` returns false without any exceptions?

Comment (KostiantynDrozd-okta) [reply direct-auth-samples/src/test/java/pages/RootPage.java]: Fixed

**Review by sergiishamrai-okta — APPROVED** [review]
LGTM

**Review by VitaliiTytarenko-okta — APPROVED** [review]

**Pull request merged**